test/browser-on-saucelabs.spec.coffee

```diff
@@ -139,7 +139,7 @@ describeBrowserTest = (browserName, getDesired, getSite) ->
             expect(err).to.be(null)
             el.text (err, text) ->
               expect(err).to.be(null)
-              expect(text).to.match(/^\s*α\s*$/)
+              expect(text).to.match(/^\s*(α|𝛼)\s*$/)
```

Comment:
Looks interesting.

Reply:
well duh, it's unicode :unicorn:

Reply:
But, get your tests to pass!

Reply:
I'm trying, it's failing on Safari on OSX 10.13 (#187).
But as I wrote in PR description, this PR fixes another problem in test that I believe doesn't happen yet, but will happen if I update MathJax.  So it's an improvements towards tests passing, and in this case I'm going to merge without all tests passing.